pkg/interface/chat/src/js/components/join.js

```diff
@@ -115,7 +115,7 @@ export class JoinScreen extends Component {
         </div>
         <h2 className="mb3 f8">Join Existing Chat</h2>
         <div className="w-100">
-          <p className="f8 lh-copy mt3 db">Enter a <span className="mono">~ship/chat-name</span> or <span className="mono">~/~ship/chat-name</span></p>
+          <p className="f8 lh-copy mt3 db">Enter a <span className="mono">~/~ship/chat-name</span></p>
```

Comment:
No, it could be either. `~ship/chat-name` are managed by groups, sig prepended paths are not.

Reply:
Ok, I'll close this out shortly then. What does `groups` mean in this case?

Reply:
Groups are what you make in the Groups app, a set of ships who share "channels", or chatrooms/notebooks/link collections, across them.

Reply:
Sig prepended path means that it's an ad hoc chat outside of any group.

Reply:
Ahhh, that makes sense. I'm just now booting my old ship back up and getting familiar with the new OS. I see Groups on the dashboard now, must've overlooked it previously. Thanks for clearing that up!